store.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default class MemoryStore {
```

Comment:
No need to create it, we already have on the wallet lib a `MemoryStore` that can be used.

Reply:
Fixed. Thanks!